es-vue-base/src/lib-utils/footer-content.js

```diff
@@ -0,0 +1,65 @@
+export default (
+    ES_DOMAIN = 'https://www.energysage.com',
+    NEWS_DOMAIN = 'https://news.energysage.com',
+    COMMUNITY_SOLAR_DOMAIN = 'https://communitysolar.energysage.com',
+    HEAT_PUMPS_DOMAIN = 'https://heatpumps.energysage.com',
+    HELP_DOMAIN = 'https://help.energysage.com',
```

Comment:
I wonder if we add `process.env` calls in here it would allow us to override the default variable at build time.

```javascript
    ES_DOMAIN = process.env.ES_DOMAIN || 'https://www.energysage.com',
```

Reply:
Good suggestion, that worked! Tested it with a `.env` file and `nuxt.config.js` pass-through in `es-design-system` and it changed the domain nicely - definitely simplifies a step in consuming applications.